extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1029,17 +974,16 @@ public function isFieldSortable( $field ) {
 	 * @param string $field: field name
 	 * @param mixed $value: field value
 	 *
-	 * @return string: formated table field
+	 * @return string: formatted table field
 	 */
 	function formatValue( $field, $value ) {
 		global $wgWikiaBaseDomain;
-		switch ($field) {
+		switch ( $field ) {
 			case "city_url":
-				preg_match("/http:\/\/([\w\.\-]+)\//", $value, $matches );
-				$sRetval = str_ireplace(".".$wgWikiaBaseDomain, "", $matches[1]);
-				return $sRetval;
+				preg_match( "/http:\/\/([\w\.-]+)\//", $value, $matches );
+				$retVal = str_ireplace( "." . $wgWikiaBaseDomain, "", $matches[1] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$retVal". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)